### PR TITLE
Update custom domains

### PR DESCRIPTION
**Pull request by johnake**
## Context
We want to make it easier to create/configure domains for services

## Changes proposed in this pull request
Update the Custom Domains area in onboard-service.md

## Checklist

- [x] I have performed a self-review of my code, including formatting and typos
- [x] I have [cleaned the commit history](https://www.annashipman.co.uk/jfdi/good-pull-requests.html)
- [x] I have added the `Devops` label
- [x] I have attached the pull request to the trello card

### CODE REVIEW

**Comment by johnake** [comment]
[Retro Action: Review and update domains doc](https://trello.com/c/v2gCUSNe/1728-retro-action-review-and-update-domains-doc)

File: documentation/onboard-service.md [hunk 1]
```diff
@@ -100,6 +100,8 @@ The code deploys integrated DNS zone and Azure front door **in the production su
 ### Deploy infrastructure
 This deploys one DNS zone and one front door which will be used subsequently by all the domains.
 
+Refer to the new service template in templates/new_service/terraform/domains/infrastructure, for the files needed before running the commands below:
```
Comment (saliceti) [inline documentation/onboard-service.md:103]: At this point, users will have already used the template script as per https://github.com/DFE-Digital/teacher-services-cloud/blob/main/documentation/onboard-service.md#generate-code
So there is no template involved anymore.

File: documentation/onboard-service.md [hunk 2]
```diff
@@ -109,9 +111,14 @@ Take note of the name servers of the newly created DNS zone and request delegati
 - [Infrastructure and networks](https://technical-guidance.education.gov.uk/infrastructure/hosting/dns/#service-now) for the education.gov.uk zone
 - [GDS](https://technical-guidance.education.gov.uk/infrastructure/hosting/dns/#official-service-gov-uk-domains) for the service.gov.uk zone
 
+### Create domain.sh in global config
```
Comment (saliceti) [inline documentation/onboard-service.md:114]: domain.sh should already be there as it was created by the template: https://github.com/DFE-Digital/teacher-services-cloud/blob/main/templates/new_service/global_config/domains.sh

File: documentation/onboard-service.md [hunk 3]
```diff
@@ -109,9 +111,14 @@ Take note of the name servers of the newly created DNS zone and request delegati
...
+In global config, create domain.sh and also update the Makefile have a make command for it
+
 ### Create custom domain per environment
 This configures the custom domain for a particular environment in the previously create DNS zone and front door.
 
+Refer to the new service template in templates/new_service/terraform/domains/environment_domains, for the files needed before running the commands below:
```
Comment (saliceti) [inline documentation/onboard-service.md:120]: At this point, users will have already used the template script as per https://github.com/DFE-Digital/teacher-services-cloud/blob/main/documentation/onboard-service.md#generate-code
So there is no template involved anymore.

File: documentation/onboard-service.md (hunk 1 above)
Comment (johnake) [reply documentation/onboard-service.md:103]: Ok, maybe I'd remove - It probably applies or help someone stating from scratch. Perhaps I could keep but add something like "Ensure the files present in /terraform/domains/infrastructure matches the sample structure in templates/new_service/terraform/domains/infrastructure, with the appropriate configuration" ?

File: documentation/onboard-service.md (hunk 2 above)
Comment (johnake) [reply documentation/onboard-service.md:114]: Same as above. Should I remove or use something like "Ensure ..."?

File: documentation/onboard-service.md (hunk 3 above)
Comment (johnake) [reply documentation/onboard-service.md:120]: Same as above. Should I remove or use something like "Ensure ..."?

File: documentation/onboard-service.md (hunk 1 above)
Comment (saliceti) [reply documentation/onboard-service.md:103]: As discussed, move to the top of the Custom domain section with more generic "Ensure..."

**Review by saliceti — APPROVED** [review]

**Pull request merged**